commands/motivation.js

```diff
@@ -0,0 +1,21 @@
+const { SlashCommandBuilder } = require("discord.js");
+const axios = require("axios").default;
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("motivation")
+    .setDescription("Get a random motivation quote."),
+  async execute(interaction) {
+    await axios({
+      method: "get",
+      url: "https://nodejs-quoteapp.herokuapp.com/quote",
+      responseType: "json",
+    })
+      .then((response) => {
+        interaction.reply("'"+response.data.quote+"'");
```

Comment:
Can you change line 15 to `interaction.reply("```" + response.data.quote + "```");`
This shows the bot reply in a code box :) 
![image](https://user-images.githubusercontent.com/41455100/193576439-64bbb3ee-8921-45ca-b145-74c1810edee4.png)

Reply:
Done